src/api/libcellml/generatorprofile.h

```diff
@@ -105,1319 +104,1359 @@ class LIBCELLML_EXPORT GeneratorProfile
+     * of a "piecewise" statement.
      */
     void setPiecewiseElseString(const std::string &piecewiseElseString);
 
     /**
-     * @brief Test if this @c GeneratorProfile has a conditional operator.
+     * @brief Test if this @c GeneratorProfile has a "conditional" operator.
      *
-     * Test if this @c GeneratorProfile has a conditional operator.
+     * Test if this @c GeneratorProfile has a "conditional" operator.
      *
-     * @return @c true if the @c GeneratorProfile has a conditional operator,
+     * @return @c true if the @c GeneratorProfile has a "conditional" operator,
      * @c false otherwise.
      */
     bool hasConditionalOperator() const;
 
     /**
-     * @brief Set whether this @c GeneratorProfile has a conditional operator.
+     * @brief Set whether this @c GeneratorProfile has a "conditional" operator.
      *
-     * Set whether this @c GeneratorProfile has a conditional operator.
+     * Set whether this @c GeneratorProfile has a "conditional" operator.
      *
      * @param hasConditionalOperator A @c bool to determine whether this
-     * @c GeneratorProfile has a conditional operator.
+     * @c GeneratorProfile has a "conditional" operator.
      */
     void setHasConditionalOperator(bool hasConditionalOperator);
 
     // Constants.
 
     /**
-     * @brief Get the @c std::string for true.
+     * @brief Get the @c std::string for "true".
      *
-     * Return the @c std::string for true.
+     * Return the @c std::string for "true".
      *
-     * @return The @c std::string for true.
+     * @return The @c std::string for "true".
      */
     std::string trueString() const;
 
     /**
-     * @brief Set the @c std::string for true.
+     * @brief Set the @c std::string for "true".
      *
-     * Set this @c std::string for true.
+     * Set this @c std::string for "true".
      *
-     * @param trueString The @c std::string to use for true.
+     * @param trueString The @c std::string to use for "true".
      */
     void setTrueString(const std::string &trueString);
 
     /**
-     * @brief Get the @c std::string for false.
+     * @brief Get the @c std::string for "false".
      *
-     * Return the @c std::string for false.
+     * Return the @c std::string for "false".
      *
-     * @return The @c std::string for false.
+     * @return The @c std::string for "false".
      */
     std::string falseString() const;
 
     /**
-     * @brief Set the @c std::string for false.
+     * @brief Set the @c std::string for "false".
      *
-     * Set this @c std::string for false.
+     * Set this @c std::string for "false".
      *
-     * @param falseString The @c std::string to use for false.
+     * @param falseString The @c std::string to use for "false".
      */
     void setFalseString(const std::string &falseString);
 
     /**
-     * @brief Get the @c std::string for exponential constant.
+     * @brief Get the @c std::string for the "exponential" constant.
```

Comment:
Can we call it something else?  Euler's number? just "e"? natural logarithm base constant?  Exponential constant doesn't quite make sense to me ... ho hum.

Reply:
Agreed. Replaced it with Euler's number in commit 3935a413.